views/user.go

```diff
@@ -51,9 +51,18 @@ func (v *Views) UsersFunc(c echo.Context) error {
 	var err error
 
 	if c.Request().Method == "POST" {
+		u, err := url.Parse("/internal/users")
+		if err != nil {
+			return echo.NewHTTPError(http.StatusBadRequest, fmt.Errorf("invalid url: %w", err))
+		}
```

Comment:
Will this ever fail? If not, `panic`ing might be entirely valid here.

Reply:
The chance of this failing would be astronomically small, if it fails then something massively wrong would have happened, I'll panic it

Reply:
(Don't tell anyone I said this, but you could get away with ignoring the error…)